reportportal.gemspec

```diff
@@ -14,8 +14,17 @@ Gem::Specification.new do |s|
   s.required_ruby_version  = '>= 2.3.0'
   s.license                = 'Apache-2.0'
 
-  s.add_dependency('rest-client', '~> 2.0')
-  s.add_dependency('rubytree', '>=0.9.3')
+  s.add_runtime_dependency('faraday', '~> 0.13')
+
+  s.add_runtime_dependency('cucumber', '~> 3.0')
```

Comment:
Only required dependencies should be declared as runtime dependencies. Only faraday and rubytree are required.

Reply:
it's single formatted for both now, so there are no assumptions

Reply:
Only required dependencies should be declared as runtime dependencies.